src/contexts/Wallet/WalletProvider.tsx

```diff
@@ -0,0 +1,72 @@
+import { Trans } from '@lingui/macro'
+import { signInKeyp } from '@usekeyp/js-sdk'
+import { LoginPortal } from '@usekeyp/ui-kit'
+import { Button, Modal } from 'antd'
+import { useWallet } from 'hooks/Wallet'
+import { useCallback, useState } from 'react'
+
+import { WalletContext } from './WalletContext'
+
+export const WalletProvider: React.FC<React.PropsWithChildren<unknown>> = ({
```

Comment:
new WalletProvider wraps the wallet connection component, allowing us to render the connect wallet UI from anywhere in the app (i.e. from any transactor instance if there's no wallet connected). previously we could just call `useWallet().connect()` which opened its own UI, but now we need a 2-step modal flow to allow a user to select a Keyp wallet or an EOA wallet

Reply:
we'll want to replace all instances of `useWallet()` with `useJBWallet()`